test/pom.xml

```diff
@@ -108,6 +108,11 @@
             <artifactId>testcontainers</artifactId>
             <optional>true</optional>
         </dependency>
+        <dependency>
+            <groupId>org.awaitility</groupId>
+            <artifactId>awaitility</artifactId>
+            <version>4.3.0</version>
```

Comment:
The version for this is in the main pom, as well as the dependency in `dependencyManagement`. So I think you can drop the version altogether.
```suggestion
```